freqtrade/strategy/interface.py

```diff
@@ -203,15 +203,17 @@ def get_signal(self, pair: str, interval: str,
         return buy, sell
 
     def should_sell(self, trade: Trade, rate: float, date: datetime, buy: bool,
-                    sell: bool) -> SellCheckTuple:
+                    sell: bool, force_stoploss=0) -> SellCheckTuple:
         """
         This function evaluate if on the condition required to trigger a sell has been reached
         if the threshold is reached and updates the trade record.
         :return: True if trade should be sold, False otherwise
         """
         current_profit = trade.calc_profit_percent(rate)
-        stoplossflag = self.stop_loss_reached(current_rate=rate, trade=trade, current_time=date,
-                                              current_profit=current_profit)
+        stoplossflag = \
```

Comment:
i think we should use implicit line breaks (within parenteses) where possible - as encouraged by PEP8 ... ;)